core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -1181,6 +1185,20 @@ public static Expression multiplyDivide(Expression e, BigDecimal multiplier,
     }
   }
 
+  /** Implementor for the {@code PERCENTILE_CONT} aggregate function. */
+  static class PercentileContImplementor extends StrictAggImplementor {
```

Comment:
If you don't plan implement Calcite's execute function. Please remove the specific implementor about PERCENTILE_CONT and PERCENTILE_DISC. This is not fully functional.

Reply:
I thought it was a requirement to add it to the aggMap. So created an empty implementor. Cleaned up.